assets/scss/pages/login-page.scss

```diff
@@ -0,0 +1,168 @@
+.c-login__section {
+    padding: 1.5rem;
+    display: flex;
+    flex-direction: column;
+    gap: 1.5rem;
+
+    @include bp(medium) {
+        padding: 3rem 5rem;
+        gap: 3rem;
+    }
+}
+
+.c-login__title {
+    color: $color-black;
+    font-family: $font-regular;
+    font-size: 2.625rem;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 1.33;
+    margin: 0;
+}
+
+.c-login__form {
+    display: flex;
+    flex-direction: column;
+    gap: 1.5rem;
+    width: 100%;
+
+    @include bp(medium) {
+        width: auto;
+    }
+
+    .c-form__field {
+        @include bp(medium) {
+            min-width: 21.37rem;
```

Comment:
@mgbelegu please avoid these "magic values". I think nobody would notice if you would make it `21rem` or `21.5rem`